packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -1652,6 +1652,39 @@ void main() {
       },
     );
 
+  testWidgets(
```

Comment:
Can you add a test for each of the other Scaffold widgets mentioned above?

Reply:
Done